diff --git a/node_modules/expo-updates/utils/build/vcs.js b/node_modules/expo-updates/utils/build/vcs.js
index 1cd23ce..7621a39 100644
--- a/node_modules/expo-updates/utils/build/vcs.js
+++ b/node_modules/expo-updates/utils/build/vcs.js
@@ -55,7 +55,7 @@ async function isGitInstalledAndConfiguredAsync() {
         if (error.code === 'ENOENT') {
             return false;
         }
-        throw error;
+        return false;
     }
     try {
         await (0, spawn_async_1.default)('git', ['rev-parse', '--show-toplevel']);
